README.md

```diff
@@ -12,27 +12,42 @@ To build the from sources follow the procedure described in
 
 [Build from sources](https://github.com/hwangsaeul/hwangsaeul.github.io/blob/master/build_from_sources.md)
 
-## Run
+*Gaeguli* requires GStreamer 1.16 or newer.
```

Comment:
We may need GStreamer 1.18 because of streamid.

Reply:
I'm a bit worried about recommending GST 1.18 because we haven't done much testing with that version. My development is virtually all done on 1.16 + patched gst-p-bad.

Also 1.18 still misses some of our srtobject patches and we don't provide any 1.18 fork with all the needed features. So encouraging others to use GStreamer 1.18 means that either some stuff won't work for them, or they'll have to figure out the needed patches themselves and apply them on top of gst-p-bad 1.18, neither of which is desirable.

Reply:
> Also 1.18 still misses some of our srtobject patches and we don't provide any 1.18 fork with all the needed features.

I got your point. okay, let's merge this as it is.